tests/unittest_brain.py

```diff
@@ -1473,7 +1473,8 @@ def test_collections_object_not_yet_subscriptable_2(self):
 
     @test_utils.require_version(minver="3.9")
     def test_collections_object_subscriptable_3(self):
-        """With python39 ByteString class of the colletions module is subscritable (but not the same class from typing module)"""
+        """With python39 ByteString class of the collections module is subscritable
```

Comment:
This also seems to be a poor translation of some other language

Reply:
```suggestion
        """With python39 the ByteString class of the collections module is subscritable
```